src/redux/actions/adminActions.js

```diff
@@ -0,0 +1,53 @@
+import {FETCH_CATEGORY,DELETE_CATEGORY,ADD_CATEGORY,UPDATE_CATEGORY} from './type';
+import axios from 'axios';
+
+export const fetchCategory =()=>{
+    return dispatch =>{
+        axios.get('https://jsonplaceholder.typicode.com/posts')
```

Comment:
@Subhang23 I dont understand from which url are you fetching/posting the categories too. I requested changes before that time you have resolved without making any change.